contrib/src/main/java/com/datatorrent/contrib/hbase/HBaseFieldValueGenerator.java

```diff
@@ -0,0 +1,46 @@
+package com.datatorrent.contrib.hbase;
+
+import java.util.List;
+
+import com.datatorrent.lib.util.FieldValueGenerator;
+import com.datatorrent.lib.util.PojoUtils;
+
+/**
+ * A {@link FieldValueGenerator} implementation for {@link HBaseFieldInfo}
```

Comment:
Can we have better comment, this is confusing, "implementation for HBaseFieldInfo"